ddtrace/opentelemetry/span.go

```diff
@@ -35,13 +37,34 @@ func (s *span) TracerProvider() oteltrace.TracerProvider        { return s.otelt
 func (s *span) AddEvent(_ string, _ ...oteltrace.EventOption)   { /*	no-op */ }
 func (s *span) RecordError(_ error, _ ...oteltrace.EventOption) { /*	no-op */ }
 
-func (s *span) SetName(name string) { s.SetOperationName(name) }
+func (s *span) SetName(name string) {
+	s.attributes[ext.SpanName] = strings.ToLower(name)
+}
 
 func (s *span) End(options ...oteltrace.SpanEndOption) {
 	if s.finished {
 		return
 	}
 	s.finished = true
+	for k, v := range s.attributes {
+		//	if we find operation.name,
+		if k == operationNameKey || k == ext.SpanName {
```

Comment:
I'm not sure we should be looking at `span.name` at all here. It was used for things like opentracing, but it's not used by OTLP, and it's not part of the RFC. Let's try to keep it simple if we can.